tests/components/totalconnect/test_config_flow.py

```diff
@@ -137,13 +137,13 @@ async def test_reauth(hass):
     )
     assert result["step_id"] == "reauth_confirm"
 
-    result = await hass.config_entries.flow.async_configure(result["flow_id"])
-    assert result["type"] == data_entry_flow.RESULT_TYPE_FORM
-    assert result["step_id"] == "reauth_confirm"
-
     with patch(
         "homeassistant.components.totalconnect.config_flow.TotalConnectClient.TotalConnectClient"
     ) as client_mock:
+        result = await hass.config_entries.flow.async_configure(result["flow_id"])
```

Comment:
Looks like this line should be removed. The next thing to do in the test after the form is shown is to pass some input to the form. Trying to continue the flow without input isn't useful.